.github/workflows/runnable_cxx.yml

```diff
@@ -72,14 +72,14 @@ jobs:
         # Note: Pattern matching is not supported so this list is quite long,
         # and brittle, as changing an msvc entry would break on OSX, for example.
         exclude:
-          # 20.04 only has g++-9 through to 11, and clang-11.0.0 through to 13.0.0
-          - { os: ubuntu-20.04, target: clang-10.0.0 }
-          - { os: ubuntu-20.04, target: clang-9.0.0 }
-          - { os: ubuntu-20.04, target: clang-8.0.0 }
-          - { os: ubuntu-20.04, target: msvc-2019 }
-          - { os: ubuntu-20.04, target: msvc-2017 }
-          - { os: ubuntu-20.04, target: msvc-2015 }
-          - { os: ubuntu-20.04, target: msvc-2013 }
+          # 22.04 only has g++-9 through to 11, and clang-11.0.0 through to 13.0.0
```

Comment:
That comment probably isn't true anymore

Reply:
Looks like some of this might not depend on the distro but on upstream blobs…